db/migrate/20230511153449_add_foreign_keys.rb

```diff
@@ -0,0 +1,12 @@
+class AddForeignKeys < ActiveRecord::Migration[7.0]
+  def change
+    
+add_reference :comments, :user, index: true, foreign_key: true
+add_reference :comments, :post, index: true, foreign_key: true
+
+add_reference :posts, :author, index: true, foreign_key: {to_table: :users}
+add_reference :likes, :user, index: true, foreign_key: true
+
+add_reference :likes, :post, index: true, foreign_key: true
+  end
+end
```

Comment:
Please note that according to the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) instead of `user`, you should have `Author_id` and `Post_id` in place of post 👍🏽

Reply:
Thank you @rotimiazeez for this review comment.I have fixed it as per the project [requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/creating_data_model_project_v1_2.md)